### PR TITLE
Scchen/vz 4507 logstash index

### PR DESCRIPTION
**Pull request by shihchang**
# Description

* remove logstash index
* remove oss flavor test 

Fixes VZ-4507

# Checklist 

As the author of this PR, I have:

- [x] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [x] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: platform-operator/helm_config/charts/verrazzano/templates/verrazzano-logging.yaml [hunk 1]
```diff
@@ -253,6 +253,18 @@ data:
...
+      enable_ruby true
+      <record>
+        @timestamp ${time}
+      </record>
+      <record>
+        target_index verrazzano-namespace-verrazzano-system
+      </record>
+    </filter>
+
     # parse sidecar stdout
```
Comment (pfmackin) [inline platform-operator/helm_config/charts/verrazzano/templates/verrazzano-logging.yaml:268]: don't you want to remove the entire logstach config?

**Review by anders-swanson — APPROVED** [review]
approved w.r.t. the test changes. I do not have context on the logstash change.

File: platform-operator/helm_config/charts/verrazzano/templates/verrazzano-logging.yaml (hunk 1 above)
Comment (shihchang) [reply platform-operator/helm_config/charts/verrazzano/templates/verrazzano-logging.yaml:268]: @pfmackin I did try "logstash_format false" and it did cause some test failures.

Comment (ayaranga) [reply platform-operator/helm_config/charts/verrazzano/templates/verrazzano-logging.yaml:268]: @shihchang did you try setting logstash_format to false (removing logstash config altogether) and setting index_name to "verrazzano-namespace-verrazzano-system"?

**Pull request merged**